# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xorg/shared-Xau.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Taken out of X.org CVS to fix nas relocation errors on e.g. powerpc and
x86-64.

  - Rene Rebe <rene@exactcode.de>

===================================================================
RCS file: /cvs/xorg/xc/lib/Xau/Imakefile,v
retrieving revision 1.2
retrieving revision 1.3
diff -u -r1.2 -r1.3
--- xc/lib/Xau/Imakefile	2004/04/23 18:43:36	1.2
+++ xc/lib/Xau/Imakefile	2004/10/12 19:13:42	1.3
@@ -37,6 +37,9 @@
 OBJS =	AuDispose.o AuFileName.o AuGetAddr.o AuGetBest.o AuLock.o \
 	AuRead.o AuUnlock.o AuWrite.o $(K5ENCOBJ)
 
+#define IncludeSharedObjectInNormalLib
+#define UnsharedLibObjCompile(options) LibObjCompile(unshared,$(CDEBUGFLAGS) $(CLIBDEBUGFLAGS) options $(PICFLAGS))
+
 #include <Library.tmpl>
 
 DependTarget()
